app/pods/publications/in-progress/in-progress-not-minister/route.js

```diff
@@ -19,7 +19,7 @@ export default class InProgressNotRoute extends Route {
   };
 
   async model(params) {
```

Comment:
async ook weg doen
Werkt het dan nog zoals verwacht ?

Reply:
manueel getest met snel en traag netwerk
In de andere route staat het er zonder.
Als je ene promise teruggeeft weet ember dat je aant loaden bent, anders wacht het gewoon tot het iets terugkrijgt